ds_judgements_public_ui/sass/includes/_govuk_overrides.scss

```
@@ -186,3 +186,90 @@
 .govuk-form-group {
   margin-bottom: 0;
 }
+
+.govuk-header {
+  border-bottom: none;
+}
+
+.govuk-header__link--homepage:hover,
+.govuk-header__link--homepage:active {
+  border-bottom: $space-1 solid $color-black;
```

Thanks for using the new variables :) but let's only use the spacing variables for spacing related CSS - as this is a sizing, let's just set this to 1rem if that's what we would like it to be.
```suggestion
  border-bottom: 0.25rem solid $color-black;
```